src/components/Film.vue

```diff
@@ -752,6 +750,7 @@ button {
 	line-height: 2.8rem;
 	border: 0;
 	background: black;
+	transition: background ease-in-out 0.25s;
```

Comment:
Also I hate that the button transitions on hover/focus now. I really liked that the site felt "snappy" in this way. Not sure if there's a way to have both

Reply:
yah maybe not. I think we can just remove this its fine

Reply:
I just removed it